src/test/java/org/jboss/xavier/integrations/route/MainRouteBuilder_DirectCalculateFlagSharedDisksTest.java

```diff
@@ -60,18 +70,29 @@ public void mainRouteBuilder_DirectCalculate_JSONGiven_ShouldReturnExpectedCalcu
         //When
         camelContext.start();
         camelContext.startRoute("flags-shared-disks");
+        camelContext.startRoute("reevaluate-workload-inventory-reports");
         String body = IOUtils.resourceToString(fileName, StandardCharsets.UTF_8, this.getClass().getClassLoader());
 
-        camelContext.createProducerTemplate().sendBodyAndHeaders("direct:flags-shared-disks", body, headers);
+        Exchange result = camelContext.createProducerTemplate().send("direct:flags-shared-disks", exchange -> {
+            exchange.getIn().setBody(body);
+            exchange.getIn().setHeaders(headers);
+        });
 
-        verify(workloadInventoryReportService).saveAll(workloadInventoryReportModels);
+        assertThat(workloadInventoryReportModels).isEqualTo(result.getIn().getBody());
```

Comment:
Since it's `assertThat(actual).isEqualTo(expected)` in order to have the right output in case of failures, the check should be
```suggestion
        assertThat(result.getIn().getBody()).isEqualTo(workloadInventoryReportModels);
```